plugins/bundle/plugin_test.go

```diff
@@ -402,7 +402,7 @@ func TestPluginOneShotDeltaBundle(t *testing.T) {
 	p2 := bundle.PatchOperation{
 		Op:    "upsert",
 		Path:  "/a/foo",
-		Value: []string{"hello", "world"},
+		Value: []interface{}{"hello", "world"},
 	}
```

Comment:
Before, this would be roundtripped though json, and it would be _turned into_ `[]interface{}` (see [playground](https://go.dev/play/p/pqyodPVAQUa)). Now, we're feeding in the proper value types.

In real life, this shouldn't matter, since when bundles are read, their `Value`s should come from a json string unmarshalled using encoding/json, so we'd only ever get `[]interface{}`, never `[]string`, there.

Reply:
The Go Playground example you included nicely illustrates your point. This change makes sense.